src/internal/config/config.go

```diff
@@ -69,6 +70,39 @@ func (c *Config) ActiveContext(errorOnNoActive bool) (string, *Context, error) {
 	return c.V2.ActiveContext, context, nil
 }
 
+// ActiveEnterpriseContext gets the active enterprise server context in the config
+func (c *Config) ActiveEnterpriseContext(errorOnNoActive bool) (string, *Context, error) {
+	if c.V2 == nil {
+		return "", nil, errors.Errorf("cannot get active context from non-v2 config")
+	}
+	if envContext, ok := os.LookupEnv(enterpriseContextEnvVar); ok {
+		context := c.V2.Contexts[envContext]
+		if context == nil {
+			return "", nil, errors.Errorf("pachctl config error: `%s` refers to a context (%q) that does not exist", contextEnvVar, envContext)
+		}
+		return envContext, context, nil
+	}
+	context := c.V2.Contexts[c.V2.ActiveEnterpriseContext]
```

Comment:
I don't think this needs to change now. But naming things context can become annoying if anyone ever needs to important the context package.